package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build:prod": "env-cmd -f environments/.env.production next build && next export",
```

Comment:
Removed `next export ` to support static paths.
https://nextjs.org/docs/advanced-features/static-html-export